omod/src/main/java/org/openmrs/module/bedmanagement/rest/resource/AdmissionLocationResource.java

```diff
@@ -214,6 +214,8 @@ public Object update(String uuid, SimpleObject propertiesToUpdate, RequestContex
 			Integer row = bedLayout.get("row");
 			Integer column = bedLayout.get("column");
 			Context.getService(BedManagementService.class).setBedLayoutForAdmissionLocation(admissionLocation, row, column);
+			Context.getService(BedManagementService.class)
+			        .removeBedLayoutOfAdmissionLocationOutSideRowColumn(admissionLocation, row, column);
```

Comment:
Here the possible exception should be caught and an ad-hoc HTTP error returned (maybe [this one](https://github.com/openmrs/openmrs-module-webservices.rest/blob/e8fa9ad1b659342948d1ec3f2f1043e13a815efd/omod-common/src/main/java/org/openmrs/module/webservices/rest/web/response/GenericRestException.java#L20)?)

At the front-end, make sure that the toast message displays the error message to the user.